src/errors/test-run/index.js

```diff
@@ -23,6 +23,19 @@ class ActionOptionErrorBase extends TestRunErrorBase {
     }
 }
 
+class ActionError extends TestRunErrorBase {
+    constructor (type) {
+        super(CATEGORY.actionError, type);
+    }
+}
+
+export class ActionSelectorError extends ActionError {
```

Comment:
I guess it should be below there all other action error are. Currently it's in base classes section